tools/debian/rules

```diff
@@ -33,7 +33,11 @@ override_dh_install:
 	mkdir -p debian/tmp/etc/pam.d
 	install -p -m 644 tools/cockpit.debian.pam debian/tmp/etc/pam.d/cockpit
 ifneq ($(SHIP_PF_API),)
-	ln -s cockpit.css.gz debian/tmp/usr/share/cockpit/base1/patternfly.css.gz
+	if [ -f debian/tmp/usr/share/cockpit/base1/cockpit.css.gz ]; then
```

Comment:
We can drop this now. We haven't updated Debian 10 any more since Debian 11 got released last year, and Ubuntu 22.04 will be released [in two days](https://discourse.ubuntu.com/t/jammy-jellyfish-release-schedule/23906), so I'll stop doing backports to 20.04 as well (and henceforth only backport to 22.04).

Reply:
Done!

Reply:
Ack, I'll send a separate PR to drop the whole SHIP_PF_API thing.